Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -716,17 +722,18 @@ - (BOOL)isCapturingStillImage {
 
 #pragma mark - Setters
 
-- (void)setCamera:(MTBCamera)camera {
```

Comment:
Sorry, didn't notice the property could be written publicly. Will address your feedback tomorrow.